rudder-by-example/src/system/manage-registries.adoc

```diff
@@ -0,0 +1,63 @@
+= Manage Windows Registry keys
+
+== Usecase
+
+Windows softwares often come with some configuration directly taken form the Windows registries.
+A clear and correct registry management can only improve the software and system management of
```

Comment:
I don't get the meaning of this sentence ("can only"?)

Maybe something like "The software and system configuration of your Windows nodes hence relies on a clear and correct registry management"